packages/kotti-ui/source/kotti-field-select/components/GenericSelectField.vue

```diff
@@ -11,20 +11,13 @@
 				:helpTextSlot="helpTextSlot"
 			>
 				<div class="kt-field-select__input-and-tags">
-					<div
+					<KtTag
 						v-for="option in visibleSelectedTags"
 						:key="option.value"
-						class="kt-field-select__tag"
```

Comment:
https://github.com/3YOURMIND/kotti/pull/694/files#diff-ed77baecf4be88164fda17914e6441cab8fb83e39b99f9ab014b8e4aef53137aR364

needs to go as well